src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -233,8 +233,8 @@ protected final void verifyWithInlineConfigParser(String filePath, String... exp
             throws Exception {
         final TestInputConfiguration testInputConfiguration =
                 InlineConfigParser.parse(filePath);
-        final DefaultConfiguration parsedConfig =
-                testInputConfiguration.createConfiguration();
+        final Configuration parsedConfig =
+                testInputConfiguration.getConfiguration();
```

Comment:
To avoid massive refactoring, try to make new method `verifyWithInlinexmlConfig`
To use method as it required.
Not all will use this method.

Reply:
Sure. Do I need to refactor all `verifyWithInlineConfigParser` calls to `verifyWithInlineXmlConfig` in every test case that comes with an xml header?